langchain4j-oracle/pom.xml

```diff
@@ -28,17 +28,10 @@
 
         <dependency>
             <groupId>com.oracle.database.jdbc</groupId>
-            <artifactId>ojdbc11</artifactId>
+            <artifactId>ojdbc8</artifactId>
             <version>${jdbc.version}</version>
         </dependency>
 
-        <dependency>
-            <groupId>com.oracle.database.jdbc</groupId>
-            <artifactId>ucp</artifactId>
-            <version>${jdbc.version}</version>
```

Comment:
Shouldn't we add UCP, and also use it in the CommonTestOperations?

Reply:
In the contributing.md document, they ask that we avoid introducing new dependencies. I replaced UCP with a simple connection pool, TestDataSource.java.